docs/installation/development.rst

```diff
@@ -103,6 +92,16 @@ Create a new Plone instance, under `Add-ons`, choose the package
 Building the Documentation
 --------------------------
 
+To build the documentation you need to have the `Enchant <http://www.abisource.com/projects/enchant/>`_ library installed on your system.
+To install Enchant library on Ubuntu::
+
+    sudo apt-get install libenchant-dev
+
+To install Enchant library on OSX using brew. Alternatively see the
```

Comment:
The sentence that starts "Alternatively" should possibly be moved after the code block.